netbout-web/src/test/java/com/netbout/dynamo/DyMessagesITCase.java

```diff
@@ -153,4 +158,32 @@ public void codeMarkdownInMessages() throws Exception {
             Matchers.equalTo("    4 leading spaces retained")
         );
     }
+    
+    /**
+     * DyMessages can throw {@code HttpException} if invoke {@code unread()}
+     * for user which is not in a bout.
+     * @throws URISyntaxException If there is some problem inside
+     * @throws IOException If there is some problem inside
+     */
+    @Test
+    public void exceptionIfBoutNotFound() throws Exception {
+        final String name = "testunread";
+        final Aliases aliases =
+            new DyBase().user(new URN("urn:test:75066")).aliases();
+        final Alias alias = aliases.add(name);
+        final Inbox inbox = alias.inbox();
+        final Bout bout = inbox.bout(inbox.start());
+        bout.messages().post("    4 leading spaces retained  ");
+        final Friends friends = bout.friends();
+        friends.kick(name);
+        final Messages messages = bout.messages();
+        try {
+            messages.unread();
+        } catch (HttpException e) {
+            MatcherAssert.assertThat(
```

Comment:
@2686747 this passes now in any case imo. If the exception is not thrown we don't have any failing assertion. You really need to use the actual `ExpectedException` functionality here.
I mean this: https://github.com/junit-team/junit4/wiki/Exception-testing